### PR TITLE
Vf valueselection

### PR DESCRIPTION
**Pull request by saurabhB-Webonise**

### CODE REVIEW

File: app/src/main/res/layout/value_descripton_dialog_fragment.xml [hunk 1]
```diff
@@ -3,89 +3,98 @@
...
-        tools:text="You take responsibility for your actions and their consequences, and you respect others who do the same." />
+        <TextView
+            android:id="@+id/titleVcTxt"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="7dp"
+            android:gravity="center_horizontal"
+            android:letterSpacing="0.05"
+            tools:text="Title"
+            android:textColor="#454545"
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/layout/value_descripton_dialog_fragment.xml:60]: use color from xml

File: app/src/main/res/layout/value_descripton_dialog_fragment.xml [hunk 2]
```diff
@@ -3,89 +3,98 @@
...
+            android:textSize="25sp"
+            android:textStyle="bold"
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/layout/value_descripton_dialog_fragment.xml:62]: user fontfamily, no need of textstyle, refer font from zeplin. Do it else where as well.

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -92,4 +92,25 @@
...
+
+        <!-- Set this to true if you want Full Screen without status bar -->
+        <item name="android:windowFullscreen">true</item>
+        <item name="android:windowIsFloating">true</item>
+
+        <!-- This is important! Don't forget to set window background -->
+        <item name="android:windowBackground">@android:color/transparent</item>
+
+
+        <!-- Additionally if you want animations when dialog opening -->
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/values/styles.xml:110]: remove this

**Review by tejasAlsi-Webonise — CHANGES_REQUESTED** [review]
check comments and use fontfamily attribute wherever you have textview.

File: app/src/main/res/layout/value_descripton_dialog_fragment.xml (hunk 1 above)
Comment (saurabhB-Webonise) [reply app/src/main/res/layout/value_descripton_dialog_fragment.xml:60]: done

File: app/src/main/res/layout/value_descripton_dialog_fragment.xml (hunk 2 above)
Comment (saurabhB-Webonise) [reply app/src/main/res/layout/value_descripton_dialog_fragment.xml:62]: done

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (saurabhB-Webonise) [reply app/src/main/res/values/styles.xml:110]: done

**Review by tejasAlsi-Webonise — APPROVED** [review]

**Pull request merged**